src/codecoverage/bot/code_coverage_bot/codecov.py

```diff
@@ -58,7 +58,8 @@ def __init__(self, repository, revision, task_name_filter, cache_root, client_id
 
         if revision is None:
             # Retrieve revision of latest codecov build
-            self.github_revision = uploader.get_latest_codecov()
+            # TODO: use HGMO
+            self.github_revision = None
```

Comment:
@marco-c Can we use HGMO directly here, or we need to find a code coverage build from Taskcluster ?

Reply:
We need to get the latest successful build, so we can probably use the backend here (with https://coverage.moz.tools/v2/latest).

Reply:
(but the github stuff can be removed altogether, you can just set self.revision = latest_successful_revision